react/MuiCozyTheme/makeOverrides.js

```diff
@@ -618,6 +618,39 @@ const makeOverrides = theme => ({
         minWidth: '.375rem'
       }
     }
```

Comment:
Can you tell us which adjustments have to be done? Can you list which props are not working anymore and how to migrate them? 

this kind of breaking change should come with a codemod.

Reply:
Pas le temps de gérer l'impact qu'il y a eu dans ActionMenu, on ne va plus faire de BC